README.md

```diff
@@ -97,116 +97,109 @@ The `plugin:regexp/all` config enables all rules. It's meant for testing, not fo
 
 ## :white_check_mark: Rules
 
-<!--RULES_SECTION_START-->
+<!-- begin auto-generated rules list -->
 
-The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) automatically fixes problems reported by rules which have a wrench :wrench: below.
-The rules with the following star :star: are included in the `plugin:regexp/recommended` config.
+💼 Configurations enabled in.\
+⚠️ Configurations set to warn in.\
+✅ Set in the `recommended` configuration.\
```

Comment:
I would like to clarify the name of the configuration in the list and in each rule document.

```suggestion
✅ Set in the `plugin:regexp/recommended` configuration.\
```

It seems that the eslint core clearly states that it is included in `"eslint:recommended"`.

https://eslint.org/docs/latest/rules/constructor-super

Reply:
I filed https://github.com/bmish/eslint-doc-generator/issues/292 to consider this as an option. See my reasoning in the issue for why I don't think it should be the default behavior. Let me know if you still want this option to be implemented.

Reply:
I also think that it should be `plugin:regexp/recommended`. Sure, it's longer, but it also makes it easier for users to copy the string and include it in their ESLint config.

Reply:
Since you both are in favor of that format, I'm open to adding an option for it. However, one alternate pattern I have seen is that the `## Configs` section in the README lists the configs with more details including how the config would be used (`plugin:regexp/recommended`) so that we don't necessarily need to mention the full details everywhere else in the docs that the config is mentioned.

Reply:
This has now been addressed using a new option: [`configFormat: 'plugin-colon-prefix-name'`](https://github.com/bmish/eslint-doc-generator#--config-format)

https://github.com/bmish/eslint-doc-generator/releases/tag/v1.3.0

Reply:
Thank you @bmish!